test/Voting.js

```diff
@@ -0,0 +1,22 @@
+import { testVotingWithSnapshots } from "./helpers/votingTestCases";
+import { ZERO_ADDRESS } from "./helpers/constants";
+
+// this is just a convenience testfile so I dont have always have to run the entire SnapshotTokentestsuite
```

Comment:
NOTE: this comment is a reminder to delete this file before merging the PR (if that would be a pr to be merged). The file only serves to accelerate the workflow.

Reply:
oh this is actually good! I was probably not clear in my task description or I just wanted to minimize your work on tests.
actually you could remove those tests from `SnapshotToken.js` and just leave this file (put it into `extras` folder and rename to `SimpleVoting.js`)